common/liquidity_amounts.gno

```diff
@@ -9,12 +13,12 @@ func liquidityAmountsGetLiquidityForAmount0(
 		sqrtRatioAX96, sqrtRatioBX96 = sqrtRatioBX96, sqrtRatioAX96
 	}
 
-	intermediate := (sqrtRatioAX96 * sqrtRatioBX96) / Q96
+	intermediate := (sqrtRatioAX96 * sqrtRatioBX96) / consts.Q96
 
 	return amount0 * intermediate / (sqrtRatioBX96 - sqrtRatioAX96)
 }
 
-func liquidityAmountsGetLiquidityForAmount1(
+func LiquidityAmountsGetLiquidityForAmount1(
```

Comment:
I think we'll have to rename it something better later.  just like truncate the `LiquidityAmount` prefix 🤔

Reply:
once we refactor our logic and if it works fantastic both on unit tests + integeration tests, then we can have time to rename things (including package path to deploy)